dxram/src/main/java/com/yahoo/ycsb/db/DXRAMProperties.java

```diff
@@ -0,0 +1,157 @@
+package com.yahoo.ycsb.db;
+
+import java.util.Properties;
+
+import com.yahoo.ycsb.Client;
+import com.yahoo.ycsb.workloads.CoreWorkload;
+import de.hhu.bsinfo.dxutils.unit.IPV4Unit;
+
+/**
+ * Wrapper class to get properties for the DXRAMClient.
+ *
+ * @author Stefan Nothaas, stefan.nothaas@hhu.de, 09.11.2018
```

Comment:
denote authorship through git commit message ([use the github feature for multiple authors if needed](https://help.github.com/en/articles/creating-a-commit-with-multiple-authors))